internal/test_helpers/course_definition.yml

```
@@ -1280,4 +1285,25 @@ stages:
+    primary_extension_slug: "history"
+    name: "History with N argument"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for `history N` to print the last N commands.
+
+  - slug: "ag6"
+    primary_extension_slug: "history"
+    name: "Limiting history entries"
+    difficulty: medium
+    description_md: |-
+      In this stage, you'll implement support for `history N` to print the last N commands.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)